src/index.js

```diff
@@ -129,6 +132,6 @@ goButton?.addEventListener('click', (e) => {
   if (inputElement.value === '') {
     errorSpan.innerHTML = 'Please enter a Pokemon Name or ID';
   } else {
-    getPokemon(inputElement.value);
+    getPokemon(inputElement.value.toLowerCase());
```

Comment:
Very nice! 

This is a pretty standard form of validation for text inputs and always worth considering. 

Another that comes to mind is removing white space from before and after the input. 

So like:

What happens when someone enters:

" Pikachu"

It has leading white space. Will your app still work?